GSA/uswds-sf-lightning-community
# Issue 44

**Summary**

Creation of the Identifier component along with the various implementations (multi-logo, spanish, no logo, taxpayer disclaimer). 

Notes added to Wiki about implementing and using the component. 

**Test plan (required)**

- [x] deploy to org with existing USWDS implementation
- [x] Create new ContentAsset
  - Once deployed, go to Files in the Salesforce app picker
  - Select Libraries from left hand pane
  - Upload new image under Asset Library folder
  - Copy the API Name of the file
  - Check "Let guest users see this asset file on public and login pages"
  - Save
- Go to community in Experience Builder
- [x] verify `boilerplateLinkNavigationLinkSetId` renders in Identifier component
- [x] verify site logo (same as in header) displays
- Footer secondary
  - [x] secondary footer section should display social media and contact center details side by side
  - [x] turn on Footer Region Toggle, social and contact center should stack
- Fill out remaining attributes in Theme Settings
  - [x] Site Domain
  - [x] Second Agency Name
  - [x] Second Agency URL
  - [x] Second Agency Logo Asset Name
 - Cycle through Identifier Types
   - [x] Spanish - compare language to Spanish example: https://designsystem.digital.gov/components/identifier/
   - [x] Multi w Logo English - both logos should display along with agency name and url
   - [x] Multi w Logo Spanish - both logos should display along with agency name and url 
   - [x] No Logo English - no logo's should display
   - [x] No Logo Spanish no logo's should display
   - [x] Taxpayer Disclaimer English - addition of Taxpayer language after Agency name/ link.
   - [x] Taxpayer Disclaimer Spanish - addition of Taxpayer language after Agency name/ link.

**Code formatting**
👍 


**Closing issues**

closes #44
